### DISCUSSION TITLE
Uninstall "CheetSheet" with homebrew and Sparkle Auto-Update Not Configured message.

### DISCUSSION THREAD

**Discussion by 14traits** [sparkle-project/Sparkle#2677] [Q&A] [answered]
So this isn't really a sparkle issue, but the developer of the app CheatSheet, who used this framework, has stopped updating it, and Homebrew recommends uninstalling it. When it's uninstalled with Homebrew, the "Auto-update not configured" message randomly pops up and tells you to "See https://sparkle-project.org/documention/ for more information." 

Could someone here point us to a plist file or something that might get updated with the apps that sparkle should be updating? It has to get the application name from somewhere in the file system. 

Here is the info on the application from homebrew, should it be needed:
https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/c/cheatsheet.rb

**Comment by zorgiepoo** [comment, marked as answer]
Anything from https://github.com/sparkle-project/Sparkle/issues/1669 help? Seems common issue with this app.

**Reply by 14traits** [reply]
@zorgiepoo  Thanks; this helped. For anyone else troubleshooting this now or in the future, killing the process or rebooting after uninstalling this program will clear this out. I tested this on my machine several times with the kill process and just the reboot. It happens every time, like clockwork, and both solutions solve the problem.
